mobile/lib/views/news/list.dart

```diff
@@ -7,7 +7,7 @@ import 'package:mobile/services/news.dart';
 import 'details.dart';
 
 Future fetchNewsItems() async {
-  final response = await http.get('http://192.168.166.72:3000/news/');
+  final response = await http.get('http://192.168.17.15:3000/news/');

```

Comment:
should we perhaps set this up to use the .env variable/

Reply:
Agreed. Will update to utilize your work incorporating .env for Dart mobile app.